llm_on_ray/inference/models/vllm/llama-2-7b-chat-hf-vllm.yaml

```diff
@@ -15,7 +15,7 @@ ipex:
   enabled: false
   precision: bf16
 model_description:
-  model_id_or_path: meta-llama/Llama-2-7b-chat-hf
-  tokenizer_name_or_path: meta-llama/Llama-2-7b-chat-hf
+  model_id_or_path: NousResearch/Llama-2-7b-chat-hf
```

Comment:
can you revert all the changes for yamls in inference/models and still use meta-llama?

Reply:
no, they are actually referenced by CI inference test cases.